src/platform/imx8/platform.c

```diff
@@ -28,6 +28,9 @@
 #include <errno.h>
 #include <stdint.h>
 
+#define I_CACHE_ATTRIBUTE	0x2524224F
+#define D_CACHE_ATTRIBUTE	0x2521224F
```

Comment:
Don't you already manage it in `imx8.x.in`, (near `_memmap_cacheattr_reset`)?

Reply:
I completely missed this one, thanks. Updating. (sure is simpler)

Reply:
Done.